app/models/forecast.rb

```diff
@@ -1,15 +1,68 @@
 require 'forecast_io'
+require 'openssl' #evil ssl removing hack for windows since windows doesn't have a root store. I am not going to lie, this is sleezy.
+
+#Hack: http://stackoverflow.com/questions/170956/how-can-i-find-which-operating-system-my-ruby-program-is-running-on Just testing.
+def isWindows?
```

Comment:
@derekriemer if we still need this then we could probably just move it to the Rake file and leave it there.